### ISSUE TITLE
Question: does codehike support line numbers rendering?

### ISSUE THREAD

**Issue by cibolog** [code-hike/codehike#95]
Cannot find it on the website or in the provided resources - does codehike has an option to show line numbers to the left of the code snippet?

**Comment by pomber** [comment]
Not yet!

**Comment by pomber** [comment]
Now you can render line numbers passing `lineNumbers: true` on the plugin options:

```js
const { remarkCodeHike } = require("@code-hike/mdx");
const theme = require("shiki/themes/material-ocean.json");

//...
    remarkPlugins: [[remarkCodeHike, { theme, lineNumbers: true }]],
//...
```

**Comment by pomber** [comment]
:rocket: Issue was released in [`v0.3.0-next.20`](https://github.com/code-hike/codehike/releases/tag/v0.3.0-next.20) :rocket:

(you had one job @githubactionsbot)
